src/main/java/org/assertj/db/navigation/PositionWithChanges.java

```diff
@@ -125,7 +125,7 @@ public E getChangesInstance(Changes changes, ChangeType changeType, String table
     }
 
     try {
-      Class clazz = Class.forName(myself.getClass().getName().replaceAll("\\$\\$.*", ""));
+      Class clazz = Class.forName(myself.getClass().getName().replaceAll("\\$.*", ""));
```

Comment:
Hi @soezen , I think we can replace the whole line by : 

`
Class clazz = unProxy(myself.getClass());
`

Probably a missing of my part during the move from cglib to bytebuddy.

Reply:
Yes, now I see that this line is also used elsewhere in the class. I have changed it as use suggested.